lib/src/http.dart

```diff
@@ -77,7 +77,7 @@ class _PubHttpClient extends http.BaseClient {
         return false;
       }
     } else {
-      if (request.url.origin != 'https://pub.dartlang.org') return false;
+      if (request.url.origin != 'https://pub.dev') return false;
```

Comment:
Needs to be confirmed that it is OK to change this

Reply:
I think this could be made to send the headers to both the old and the new site (they are the same site behind the dns lookup). Just in case someone hardcodes PUB_HOSTED_URL to pub.dev

```
 // We recognize both the old and new name of the server.
 if (!['https://pub.dev', 'https://pub.dartlang.org'].contains(request.url.origin)) return false;
```
@jonasfj do you agree?

Reply:
Sure, that's fine.